### ISSUE TITLE
Suggestion of training epochs?

### ISSUE THREAD

**Issue by zjxgithub** [mindslab-ai/faceshifter#4]
Thanks for the implementation! Well done!
I am about to use your code to train my own model. I am curious how long it took your model to be trained? How many epochs did you use?

Thanks!

**Comment by usingcolor** [comment]
Hi! According to the paper, they trained 500K steps for AEI-Net. 
In our case, the training loss seems to be little bit random and unstable, but in the long term the loss was going down. We trained for over 500K with 32 batch size.
 For conclusion, there are no absolute training epoch, you should observe the loss graph and validation results. Then stop when you satisfy. I know it sounds little irresponsible, but this is a behavior of GAN.

**Comment by zjxgithub** [comment]
Thanks for your response! That is helpful to know!
May I ask how long it took to finish 500K steps on single GPU? Several
days? Cuz I am trying to estimate when I can finish the training.

**Comment by usingcolor** [comment]
Depends on your environment. Two V100 32G with 32 batch size took more than a week.

**Comment by zjxgithub** [comment]
I see... Thanks!

**Comment by y-x-c** [comment]
hi @usingcolor ! I wonder how many images were used in your training/validation?  thanks!

**Comment by usingcolor** [comment]
> hi @usingcolor ! I wonder how many images were used in your training/validation? thanks!
please make a new issue next time.

For the Question,
I used the dataset that I mentioned in README.md (i.e. FFHQ, CeleabA-HQ and VGG face)
You can add the face images as you want.
